packages/apps/composer-app/src/main.tsx

```diff
@@ -25,6 +25,9 @@ import MapMeta from '@braneframe/plugin-map/meta';
 import MarkdownMeta from '@braneframe/plugin-markdown/meta';
 import MermaidMeta from '@braneframe/plugin-mermaid/meta';
 import MetadataMeta from '@braneframe/plugin-metadata/meta';
+
+import NativeMeta from '../../plugins/plugin-native/src/meta';
```

Comment:
make sure to add `@braneframe/plugin-native` as dep to composer and import from that as well